.cirrus.yml

```diff
@@ -1,133 +1,186 @@
+    cpu: 4
+    memory: 12
+
+  timeout_in: 20m
+
+  script:
+    - export PATH="$PATH:$GOPATH/bin"
+    - "$SCRIPT_BASE/enable_ssh.sh"
+    - go get github.com/vbatts/git-validation
+    - make
+    - make validate
 
-    # see bors.toml
-    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+gating_task:
+  name: "Gating test"
+  alias: gating
 
-    container: &smallcontainer
-        image: "quay.io/libpod/imgts:$IMAGE_SUFFIX"
-        cpu: 1
-        memory: 1
+  # Only run this on PRs, never during post-merge testing.  This is also required
+  # for proper setting of EPOCH_TEST_COMMIT value, required by validation tools.
+  only_if: $CIRRUS_PR != ""
 
-    env:
-        IMGNAMES: >-
-            ${FEDORA_CACHE_IMAGE_NAME}
-            ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
-            ${UBUNTU_CACHE_IMAGE_NAME}
-        BUILDID: "${CIRRUS_BUILD_ID}"
-        REPOREF: "${CIRRUS_REPO_NAME}"
-        GCPJSON: ENCRYPTED[e8a53772eff6e86bf6b99107b6e6ee3216e2ca00c36252ae3bd8cb29d9b903ffb2e1a1322ea810ca251b04f833b8f8d9]
-        GCPNAME: ENCRYPTED[fb878daf188d35c2ed356dc777267d99b59863ff3abf0c41199d562fca50ba0668fdb0d87e109c9eaa2a635d2825feed]
-        GCPPROJECT: "libpod-218412"
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
 
-    clone_script: &noop mkdir -p $CIRRUS_WORKING_DIR
-    script: /usr/local/bin/entrypoint.sh
+  # Runs within Cirrus's "community cluster"
+  container:
+    image: "$FEDORA_CONTAINER_FQIN"
+    cpu: 4
+    memory: 12
 
+  timeout_in: 20m
 
-gating_task:
-    name: "Gating test"
-    alias: gating
+  depends_on:
+    - build
+
+  script:
+    - "$SCRIPT_BASE/enable_podman.sh"
+    - make validate
 
-    # Only run this on PRs, never during post-merge testing.  This is also required
-    # for proper setting of EPOCH_TEST_COMMIT value, required by validation tools.
-    only_if: $CIRRUS_PR != ""
+test_task:
+  name: "Test on $FEDORA_NAME"
+  alias: test
 
-    # see bors.toml
-    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
 
-    # Runs within Cirrus's "community cluster"
-    container:
-        image: "$FEDORA_CONTAINER_FQIN"
-        cpu: 4
-        memory: 12
+  depends_on:
+    - gating
 
-    timeout_in: 20m
+  script:
+    - "$SCRIPT_BASE/test.sh"
 
-    script:
-        - export PATH="$PATH:$GOPATH/bin"
-        - make
-        - go get github.com/vbatts/git-validation
-        - make validate
+latest_task:
+  name: "Podman latest test"
+  alias: podman_latest
 
-test_task:
-    name: "Test on $FEDORA_NAME"
-    alias: test
+  # Only run this on PRs, never during post-merge testing.  This is also required
+  # for proper setting of EPOCH_TEST_COMMIT value, required by validation tools.
+  only_if: $CIRRUS_PR != ""
+
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
 
-    # see bors.toml
-    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+  # Runs within Cirrus's "community cluster"
+  container:
+    image: "$FEDORA_CONTAINER_FQIN"
+    cpu: 4
+    memory: 12
```

Comment:
Do you really want to run this in a container?  I think you're `enable_podman.sh` will fail due to lack of systemd conditions.

Reply:
How were we running before? enable_podman.sh was extracted from latest_podman.sh so I could call it from two places.

Reply:
***Not*** specifying a `container` or `gce_instance` attribute will cause it to inherit the top-level value (which is a `gce_instance` a.k.a. a VM IIRC)